src/component.py

```diff
@@ -54,13 +55,16 @@ def run(self) -> None:
         # Move files from data/in/files to data/out/files
         self.move_files()
 
-        # Find tables that are not specified to be anonymized
-        # and move them to data/out/tables along with their manifest files
-        self.move_non_anonymized_tables(tables_to_anonymize)
+        for table in self.get_input_tables_definitions():
+            anonymize = [pattern for pattern in tables_to_anonymize.keys() if fnmatch.fnmatch(table.name, pattern)]
 
-        for table_name in tables_to_anonymize:
-            columns_to_anonymize = tables_to_anonymize.get(table_name)
-            self.anonymize_table(table_name, columns_to_anonymize, salt, salt_location)
+            if anonymize:
+                columns_to_anonymize = tables_to_anonymize.get(table.name) or tables_to_anonymize.get(anonymize[0])
```

Comment:
tady v tom pripade je ten matchnutej key vzdycky v `anonymize[0]`. tak je to or zbytecny. Taky kdyz si reknem ze ten match pattern muze byt jen jeden, tak nema smysl, aby to bylo pole.

Reply:
kdyby chtěli u tabulky definované celým jménem jiné chování než např. u všech ostatních přestože by odpovídala wildcard

Reply:
ja mel za to ze `fnmatch.fnmatch(table.name, pattern)` bude true pokud je jako pattern cely nazev tabulky

Reply:
chapu, to je nekonzistentni s nasim pravidlem, ze 1 tabulka nesmi mit vic jak 1 match. Ted je to ok za me ok ;)